fdbserver/DataDistribution.actor.cpp

```diff
@@ -3181,7 +3181,7 @@ ACTOR Future<Void> zeroServerLeftLogger_impl(DDTeamCollection* self, Reference<T
 
 	for (auto const& shard : shards) {
 		sizes.emplace_back(brokenPromiseToNever(self->getShardMetrics.getReply(GetMetricsRequest(shard))));
-		TraceEvent(SevWarnAlways, "DDShardLost", self->distributorId)
+		TraceEvent(SevError, "DDShardLost", self->distributorId)
```

Comment:
From the discussion with A.J., let's change it back to SevWarnAlways for now. 
SRE will search Sev30 for incident diagnosis and filter the noisy events manually unfortunately.

Reply:
I think you raise some good points, though, so we shouldn't let this idea drop completely. Not sure how we want to track or plan a solution to this, but maybe we can record a generic GitHub issue for it?